lcls-twincat-pmps/PMPS/MajorComponents/BPTM/BeamParameterTransitionManager.TcPOU

```diff
@@ -8,6 +8,7 @@ NOTE:
 The BPTM will throw an error if the arbiter does not have enough space for the transition and new final assertion. 
 
  *)
+{attribute 'no_check'}
```

Comment:
Should we revisit these "no check" attributes later? It's a little bit concerning that they were all needed.